packages/x-codemod/src/v6.0.0/pickers/rename-should-disable-time/actual-props.spec.js

```diff
@@ -0,0 +1,5 @@
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+
+<div>
+  <DateTimePicker shouldDisableTime={(timeValue, view) => view === 'hours' && timeValue < 12} />
```

Comment:
We could also introduce a test case with function as a variable just for the sake wider coverage. 🙈 

```diff
diff --git a/packages/x-codemod/src/v6.0.0/pickers/rename-should-disable-time/actual-props.spec.js b/packages/x-codemod/src/v6.0.0/pickers/rename-should-disable-time/actual-props.spec.js
index 898616043..bf32e31b8 100644
--- a/packages/x-codemod/src/v6.0.0/pickers/rename-should-disable-time/actual-props.spec.js
+++ b/packages/x-codemod/src/v6.0.0/pickers/rename-should-disable-time/actual-props.spec.js
@@ -1,5 +1,9 @@
 import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import { TimePicker } from '@mui/x-date-pickers/TimePicker';
+
+const shouldDisable = (timeValue, view) => view === 'hours' && timeValue < 12;
 
 <div>
   <DateTimePicker shouldDisableTime={(timeValue, view) => view === 'hours' && timeValue < 12} />
+  <TimePicker shouldDisableTime={shouldDisable} />
 </div>;
diff --git a/packages/x-codemod/src/v6.0.0/pickers/rename-should-disable-time/expected-props.spec.js b/packages/x-codemod/src/v6.0.0/pickers/rename-should-disable-time/expected-props.spec.js
index 784d823db..f3e13aeaf 100644
--- a/packages/x-codemod/src/v6.0.0/pickers/rename-should-disable-time/expected-props.spec.js
+++ b/packages/x-codemod/src/v6.0.0/pickers/rename-should-disable-time/expected-props.spec.js
@@ -1,5 +1,9 @@
 import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import { TimePicker } from '@mui/x-date-pickers/TimePicker';
+
+const shouldDisable = (timeValue, view) => view === 'hours' && timeValue < 12;
 
 <div>
   <DateTimePicker shouldDisableClock={(timeValue, view) => view === 'hours' && timeValue < 12} />
+  <TimePicker shouldDisableClock={shouldDisable} />
 </div>;

```

Reply:
If you feel safer with it, I add it with pleasure :)